clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -160,6 +224,11 @@ export default async function handler(
     // Include generic fides.js script
     ${fidesJS}
 
+    // Include custom-fides.css
+    const style = document.createElement('style');
+    style.innerHTML = ${JSON.stringify(customFidesCss)};
+    document.head.append(style);
```

Comment:
We should put this in a guard like `if (customFidesCss is not an empty string)`, otherwise I think this'll do some strange things when no custom CSS exists

Reply:
I tested this use case and in the event of an empty custom stylesheet `sytle.innerHTML` will just be assigned `""`. No side-effects reported.

Reply:
Updated the logic to conditionally include the custom styles